c7n/resources/account.py

```diff
@@ -1243,3 +1243,69 @@ def process(self, resources):
             client.put_public_access_block(
                 AccountId=r['account_id'],
                 PublicAccessBlockConfiguration=config)
+
+
+@actions.register('set-glue-encryption')
+class SetGlueEncryption(BaseAction):
+    """Set Glue encryption settings (dataCatalog and connection pws)
+    :example:
+    .. code-block:: yaml
+            policies:
+              - name: set-datacatalog-encryption
+                resource: aws.account
+                actions:
+                  - type: set-glue-encryption
+                    DataCatalogEncryptionSettings
+                      EncryptionAtRest:
+                        CatalogEncryptionMode: SSE-KMS
+                        SseAwsKmsKeyId: arn:aws:kms:us-east-1:<acc_num>:key/<key_id>
```

Comment:
use the alias here since its supported by the action

Reply:
Sure. Fixed it. Please review.